api/types.proto

```diff
@@ -277,6 +277,41 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	enum OnFailure {
+		PAUSE = 0;
```

Comment:
Last change wins.

This is how you manually trigger a rollback

On Thursday, July 21, 2016, Aaron Lehmann <notifications@github.com> wrote:

> In api/types.proto
> <https://github.com/docker/swarmkit/pull/1204#discussion_r71820830>:
>
> > @@ -277,6 +277,41 @@ message UpdateConfig {
> >
> >  	// Amount of time between updates.
> >  	Duration delay = 2 [(gogoproto.nullable) = false];
> > +
> > +	enum OnFailure {
> > +		PAUSE = 0;
>
> The current behavior is that a change to the service replaces the previous
> rollout. This PR doesn't change that aspect.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/swarmkit/pull/1204/files/ade92c98a9729983856b15327850b3f324a65eec#r71820830>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAK9LzSs_btn7j8A1vHARmtQ3Iv3VeG9ks5qYDJbgaJpZM4JSIbd>
> .
>